### PR TITLE
change: [M3-7735] - Improve Linode Graph X Axis Labels when viewing historic data

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

Updates the Linode Graph's X Axis to show **days** instead of **hours** on the x-axis when the user select a longer time period 📈

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-02-13 at 9 45 31 AM | Screenshot 2024-02-13 at 9 45 21 AM |

### Prerequisites
- Make sure you have the `recharts` flag on

### Verification steps 
- Go to the Analytics tab of on a Linode's details page
- Try viewing the last 30 days or longer of stats
- Verify the X Axis is showing units in days instead of hours
- Verify the X Axis is user friendly

## As an Author I have considered 🤔

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [x] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [x] 📱 Providing mobile support
- [x] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx [hunk 1]
```diff
@@ -115,6 +115,8 @@ const LinodeSummary: React.FC<Props> = (props) => {
     debouncedRefetchLinodeStats();
   }, [windowWidth, windowHeight, debouncedRefetchLinodeStats]);
 
+  const xAxisTickFormat = isLast24Hours ? 'hh a' : 'LLL dd';
```
Comment (mjac0bs) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx:118]: Thanks for the const here. Readability would be further improved with a short comment about what the tick formats actually mean.

**Review by mjac0bs — COMMENTED** [review]
Confirmed that I am seeing days instead of hours on the x-axis now when looking at historical (Last 30 Days) data, but the title of our internal ticket does mention fixing a discrepancy with units in the tooltip.

I am still seeing that for the Network graph with `b/s` and units are in `kb/s`. 
Screenshot 2024-02-13 at 2 08 44 PM

**Comment by bnussman-akamai** [comment]
A seperate ticket M3-7758 was created for the networking units issue. I fixed the title of M3-7735!

**Review by abailly-akamai — APPROVED** [review]
Thanks! change verified

should Disk I/O show `b/s` in the legend (every other graph there shows the unit after the value
Screenshot 2024-02-14 at 10 07 57

**Review by hana-linode — APPROVED** [review]
Verified X Axis labels show days when a time period is greater than 24 hours 📈

**Review by mjac0bs — APPROVED** [review]

**Pull request merged**